File: php/uber-type/uber-black.php
require_once('car.php);
class UberBlack extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function _construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
        parent::_construct($license, $driver)
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

File: python/car-type/uber-black.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    public __init__(self, license, driver, typeCarAccepted, seatsMaterial)
        super__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

